### ISSUE TITLE
2nd level test pruning.py fails with "insufficient priority"

### ISSUE THREAD

**Issue by error10** [bitcoin/bitcoin#6834]
In Bitcoin Core 0.11.1, this test script fails with `JSONRPC error: 66: insufficient priority` as shown below. The test passed in 0.11.0.

```
Running 2nd level testscript pruning.py...
  File "/builddir/build/BUILD/bitcoin-0.11.1/qa/rpc-tests/test_framework/test_framework.py", line 118, in main
    self.run_test()
  File "/builddir/build/BUILD/bitcoin-0.11.1/qa/rpc-tests/pruning.py", line 261, in run_test
    self.create_big_chain()
  File "/builddir/build/BUILD/bitcoin-0.11.1/qa/rpc-tests/pruning.py", line 76, in create_big_chain
    self.mine_full_block(self.nodes[0], self.address[0])
  File "/builddir/build/BUILD/bitcoin-0.11.1/qa/rpc-tests/pruning.py", line 253, in mine_full_block
    txid = node.sendrawtransaction(signresult["hex"], True)
  File "/builddir/build/BUILD/bitcoin-0.11.1/qa/rpc-tests/test_framework/authproxy.py", line 126, in __call__
    raise JSONRPCException(response['error'])
Initializing test directory /tmp/testM1XDl2
Warning! This test requires 4GB of disk space and takes over 30 mins (up to 2 hours)
Mining a big blockchain of 995 blocks
JSONRPC error: 66: insufficient priority
Stopping nodes
Cleaning up
Failed
```

**Comment by morcos** [comment]
This is due to the min relay fee going up.

**Comment by morcos** [comment]
Fixed with #6841
